marlowe-runtime/tx/Language/Marlowe/Runtime/Transaction.hs

```diff
@@ -7,47 +7,177 @@ module Language.Marlowe.Runtime.Transaction
   where
```

Comment:
Changes in this file:

- Move server sources to component outputs
- Connection refactor
- Move span rendering from Logging.hs